plugins/org.komodo.relational/src/org/komodo/relational/internal/RelationalChildRestrictedObject.java

```diff
@@ -33,6 +34,16 @@ protected RelationalChildRestrictedObject( final UnitOfWork uow,
     /**
      * {@inheritDoc}
      *
+     * @see org.komodo.repository.ObjectImpl#getChildTypes()
+     */
+    @Override
+    public final KomodoType[] getChildTypes() {
+        return KomodoType.NO_TYPES;
+    }
```

Comment:
Yes the name is intended to convey that. And the implementation of that class contains child-related methods that all have a `final` modifier to try and enforce the no-child rule. I am open to other class names if you have a better one.